app/components/GameHeader/GameHeader.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import Instructions from '../Instructions/Instructions';
+
+export default function GameHeader({ state, }) {
+  const settingsText = '<-- Settings';
+  const {
+    pathStartingPoint,
+    pathEndingPoint,
+    startButtonPressed,
+    endButtonPressed,
+  } = state;
```

Comment:
Destructure in argument

Reply:
Also 'state' in `props`, while `state` is also a common thing in React is very confusing. I'd rename.

Reply:
what do you think about `appState`? 

(`relevantAppState` sounds wrong..)

Reply:
gameState?

Reply:
decided to give every component's state its separate state name.

let me know if it seems too much and perhaps `gameState` for all would look better